pythonx/completor/__init__.py

```diff
@@ -92,7 +92,7 @@ def get_option(key):
     @property
     def disabled(self):
         types = vim.vars.get('completor_disable_{}'.format(self.filetype))
-        if isinstance(types, (int, long)):
+        if isinstance(types, (int, int)):
```

Comment:
Integer types in py2 and py3 are different. Both `int` and `long` are in py2, but only `int` in py3. So I think to fix this problem we should consider both situation and add this:
```python
import sys
integer_types = (int, long) if sys.version_info[0] == 2 else (int,)
```
and change this line to `if isinstance(types, integer_types)`.